db/schema.rb

```diff
@@ -70,6 +66,15 @@
     t.index ["person_id"], name: "index_educations_on_person_id"
   end
 
+  create_table "employee_quantities", force: :cascade do |t|
+    t.string "category"
+    t.integer "quantity"
+    t.bigint "company_id"
+    t.datetime "created_at", null: false
```

Comment:
isn't there t.timestamps ?

Reply:
should this be generated automatically?

Reply:
yes, the schema.rb is for example updated when you run rake db:migrate
make sure the newest version of schema.rb is checked in.

Reply:
In the schema.rb all entities have a created_at and updated_at timestamp with datatype t.datetime, there are no t.timestamps. I'm not quite sure, what is false?

Reply:
there's nothing wrong ... it's just rails default to use t.timestamps
http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-timestamps
you then have only one line :)